src/views/Help/help.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../vars';
+
+ul{
```

Comment:
Can we get rid of these styles, now that we are relying on `Typography` instead?